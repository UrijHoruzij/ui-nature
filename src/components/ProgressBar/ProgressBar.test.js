import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { ProgressBar } from "../";
import ThemeProvider from "../../ThemeProvide";

describe("ProgressBar component", () => {
  test("Render progressbar", () => {
    const component = (
      <ThemeProvider>
        <ProgressBar percent={78} />
        <ProgressBar percent={45} type="error" />
      </ThemeProvider>
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
