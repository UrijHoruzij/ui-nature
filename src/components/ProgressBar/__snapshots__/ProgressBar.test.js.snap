// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar component Render progressbar 1`] = `
Array [
  .c0 {
  position: relative;
  height: 4px;
  width: 100px;
  border-radius: 4px;
  background: #f0f0f0;
}

.c1 {
  position: absolute;
  top: 0;
  left: 0;
  width: 78%;
  height: 100%;
  background: #008603;
  border-radius: 4px;
}

<div
    className="c0"
    type="success"
  >
    <div
      className="c1"
      type="success"
    />
  </div>,
  .c0 {
  position: relative;
  height: 4px;
  width: 100px;
  border-radius: 4px;
  background: #f0f0f0;
}

.c1 {
  position: absolute;
  top: 0;
  left: 0;
  width: 45%;
  height: 100%;
  background: #f3363e;
  border-radius: 4px;
}

<div
    className="c0"
    type="error"
  >
    <div
      className="c1"
      type="error"
    />
  </div>,
]
`;
